designs/bee/src/band-tie.mjs

```diff
@@ -26,131 +27,125 @@ export const bandTie = {
     paths,
     options,
     complete,
+    paperless,
     macro,
     measurements,
-    snippets,
-    expand,
-    Snippet,
-    utils,
     absoluteOptions,
     part,
   }) => {
-    if (!options.ties && !options.crossBackTies) part.hide()
-
-    const bandTieLength = options.crossBackTies
-      ? (measurements.underbust * options.bandLength) / 2 + options.neckTieWidth * 2
-      : (measurements.underbust + measurements.underbust * options.bandTieLength) / 2
-    const bandTieWidth = options.crossBackTies
-      ? absoluteOptions.bandTieWidth * 2
-      : absoluteOptions.bandTieWidth
-
-    /*
-     * Don't bother unless expand is set
-     */
-    if (!expand) {
-      points.text = new Point(10, 10)
-        .addText('bee:cutBandTie', 'fill-note')
-        .addText(':')
-        .addText(utils.units((bandTieWidth + sa) * 2))
-        .addText(' x ')
-        .addText(utils.units(bandTieLength))
-      paths.diag = new Path().move(new Point(0, 0)).line(new Point(100, 15)).addClass('hidden')
-
-      return part
+    //set Render
+    if (!options.ties && !options.crossBackTies) {
+      return part.hide()
+    }
+    //measures
+    let bandTieLength
```

Comment:
This was rewritten as a const with ternary operator.
I'm not sure if this is an opinionated rollback, or if this PR just contains outdated code?